x-pack/plugins/maps/common/descriptor_types/layer_descriptor_types.ts

```diff
@@ -29,6 +34,7 @@ export type LayerDescriptor = {
   __trackedLayerDescriptor?: LayerDescriptor;
   __areTilesLoaded?: boolean;
   alpha?: number;
+  attribution?: Attribution;
```

Comment:
should we remove, if only since we're not using it?

Reply:
Not sure what you mean, layer.attribution is getting used. This is where its store now as opposed to layer.sourceDescriptor.

Reply:
my mistake, I misread this. sorry for the noise.